loops_lab.js

```diff
@@ -0,0 +1,128 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+//
+let num = 15;
+
+while (num >= 1){
+  console.log(num);
+  num -= 1;
+}
+
+console.log("*****");
+
+for(let x = 15; x >=1; x -=1){
+  console.log(x);
+}
+
+console.log("-------------  ");
+//
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+let  max = 0
+
+while (max <= 40){
+  max += 1
+  if( max % 2 === 1 ){
+    console.log(max + " is odd");
+  } else {
+    console.log(max + " is even ");
+  }
+
+}
+
+console.log("*****");
+
+for(let max2 = 1; max2 <= 40; max2 += 1){
+  if( max2 % 2 === 1 ){
+    console.log(max2 + " is odd");
+  } else {
+    console.log(max2 + " is even ");
+  }
+}
+
+console.log("--------------");
+
+// Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//     <details>
+//       <summary>
+//         Hint
+//       </summary>
+//       Find the final number and increment the loop by 9.
+//     </details>
+
+  x = 1
+  while (x <=10){
+      console.log( x + " * 9 = " + (x * 9));
+      x++
+}
+
+console.log("*****");
+
+for(let y = 1 ; y <= 10; y++){
+    console.log( y + " * 9 = " + (y * 9) );
+  }
+
+console.log("--------------");
+
+// Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+//
+let a = 0
```

Comment:
Creative way of solving the problem but this would be considered hard coding in some way. Can you think of a way where you would use modulo to solve this problem? i.e. a % 5 && a % 10